helpers/colour-contrast/rgb.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * A class to parse color values
+ * @author Stoyan Stefanov <sstoo@gmail.com>
+ * @link   http://www.phpied.com/rgb-color-parser-in-javascript/
+ * @license MIT license
+ */
```

Comment:
License-wise, wondering about the implications of including this here inside something of ours that's Apache 2.0?

If we could restrict this to HEX-only values (which I believe is the most common), then I think it could be dramatically simplified... we'd just take the 6-character string, split it into three 2-character strings and run `parseInt(val, 16)` on each one. Could probably do this whole thing in a few lines.